.github/workflows/gotests.yml

```diff
@@ -96,3 +96,26 @@ jobs:
         name: btf-vmlinux-${{ matrix.os }}
         path: /sys/kernel/btf/vmlinux
         retention-days: 5
+
+  build_windows:
+    runs-on: windows-latest
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5.3.0
+      with:
+        # renovate: datasource=golang-version depName=go
+        go-version: '1.24.2'
+
+    - name: Checkout code
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        path: go/src/github.com/cilium/tetragon/
+
+    
+    - name: Run go tests
+      working-directory: ${{ github.workspace }}\go\src\github.com\cilium\tetragon
+      run: |
+        go test --failfast .\pkg\... .\cmd\...
```

Comment:
but can you reuse the existing job workflow instead a new job? you can use the os matrix and skip some steps if they are running on windows

Reply:
The unit test workflow in Linux runs via make test. There is no makefile in Windows.

Reply:
ah yeah okay I see